### PR TITLE
C++ Fathead: Add language keywords 

### LINKED ISSUE #573: C++: List all topic titles and language features to help measure coverage
### Description
For every Fathead, we need to gather the set of articles we wish to provide coverage for.
Create a `cover/` directory containing text files that list all the topic titles and language features to cover. 
These lists will then be used as Unit tests to help you measure the Fathead's coverage. 
For more info please see [the docs](https://docs.duckduckhack.com/programming-mission/creating-effective-fatheads.html).

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/cppreference_doc

### PR DESCRIPTION
**Pull request by ankushg07**
## Description of new Instant Answer, or changes

Addition of keywords used in c++ along with some built in functions names 

## Related Issues and Discussions
Fixes #573 

## People to notify
@tagawa 
## Testing & Review
To be completed by Language Leader (or DDG Staff)

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page**
- [x] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)
- [x] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Code**
- [x] Uniformly indented, well commented
- [x] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/cppreference_doc

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ankushg07 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by tagawa** [comment]
Hi @ankushg07. Thank you for this fix. Please could you change the name of the file to `keywords.txt`? Then it should be picked up by the automatic redirect script.

**Comment by ankushg07** [comment]
@tagawa any other issue sir?

**Comment by tagawa** [comment]
Thank you for the changes @ankushg07 - it looks good to me. 
@moollaza Do you agree we can merge this?

File: lib/fathead/cppreference_doc/cover/keywords.txt [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+virtual
+void
+volatile
+vprintf
+vsprintf
+wchar_t
+while
+xor
+xor_eq
+
```
Comment (sahildua2305) [inline lib/fathead/cppreference_doc/cover/keywords.txt]: Looks good!
Minor Minor Nit: Remove this blank line

**Comment by ankushg07** [comment]
thanks @sahildua2305

**Pull request merged**